src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -222,23 +228,18 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
           value, // Amount
           transactionInHex: await rawTransactionBody.text(),
         };
-      });
+      }));
 
-    const utxoDetails = await Promise.all(fetchHexPromises);
-
-    const amountInSatoshi = toSatoshi(amountInBtc);
-    const feeInSatoshi = toSatoshi(options.fee);
-    let totalBalance: number = 0;
-    let hasSufficientBalance = false;
+    // Sort UTXOs by ascending absolute difference from the amount to send
+    const spendableUtxos = fullUtxos.sort(
+      (a, b) => Math.abs(a.value - (amountInSatoshi + feeInSatoshi))
+              - Math.abs(b.value - amountInSatoshi + feeInSatoshi),
```

Comment:
I understand that it looks nice to have the `Math.abs` aligned in one column. But aligning stuff to columns in longer term is rather bad practice - imagine everyone will be aligning stuff this way :) Also with this approach when one line changes it's indentation you need to align all others, which leads to more affected lines than it's needed.
```suggestion
      (a, b) => (
        Math.abs(a.value - (amountInSatoshi + feeInSatoshi))
        - Math.abs(b.value - amountInSatoshi + feeInSatoshi)
      ),
```